src/Sylius/Bundle/ApiBundle/Entity/Product/Product.php

```diff
@@ -13,44 +13,65 @@
 class Product extends CoreProduct implements ProductInterface
 {
     /**
-     * @var int
+     * @var int|null
      */
     private $onHand;
 
     /**
-     * @return int
+     * @var int|null
+     */
+    private $minPrice;
+
+    /**
+     * @var int|null
+     */
+    private $maxPrice;
+
+    /**
+     * @return int|null
```

Comment:
This is an unnecessary comment, everything is already clear by method signature

Reply:
I see, my editor is getting nuts when auto-creating-methods).
But there's a reason for doing it to, as we're not using the latest version of ApiPlatform
To a certain version reading types from annotations was better optimized than doing it from method-signature at least for this issue we might go this way;

Reply:
...it refers to getters, setters and properties that will be handled as Resource properties

Reply:
if you'll insist on removing all the annotations, I'll do it

Reply:
I think there can't be more performance with annotation. But anyway we are using `friendsofphp/php-cs-fixer` on our projects, so install it on sylius with --dev param
also you may configure it with autoworking with this params 


-------------------------

## Configure PHP CS Fixer

* PhpStorm

    1. Open **Settings > Tools > File Watchers**
    2. Create Watcher with these settings:

        * **File type:** PHP
        * **Scope:** Current File
        * **Program:** `$ProjectFileDir$/vendor/bin/php-cs-fixer`
        * **Arguments:** `fix --allow-risky=yes $FilePathRelativeToProjectRoot$`
        * **Working directory:** `$ProjectFileDir$`

        Advanced Options: uncheck everything


-------------------
Otherwise you may use it manually before each commit. Or configure your IDE ;)